src/components/TxHistory/TxHistoryListItem.js

```diff
@@ -83,37 +89,64 @@ const messages = defineMessages({
     defaultMessage: '!!!Failed',
     description: 'some desc',
   },
+  assets: {
+    id: 'global.txLabels.assets',
+    defaultMessage: '!!!{cnt} assets',
+    description: 'The number of assets different assets, not the amount',
+  },
+})
+
+const ASSURANCE_MESSAGES: $ReadOnly<Dict<any>> = Object.freeze({
```

Comment:
I think this can be of type `Dict<MessageDescriptor>` (we added `intl` types recently)